.rubocop.yml

```diff
@@ -20,9 +24,27 @@ Metrics/BlockLength:
     - 'spec/**/*.rb'
     - 'openfeature-sdk.gemspec'
 
+Gemspec/DevelopmentDependencies:
+  EnforcedStyle: gemspec
+
 Gemspec/RequireMFA:
   Enabled: false
 
 Style/DocumentDynamicEvalDefinition:
   # TODO re-enable after figuring out what it actually wants
   Enabled: false
+
+RSpec/ContextWording:
+  Enabled: false
+
+RSpec/MultipleExpectations:
+  Max: 2
```

Comment:
I always find the single default expecation max as overly punitive; I think 2 is a much more reasonable default.